libs/sdk-ui-dashboard/.dependency-cruiser.js

```diff
@@ -16,7 +16,7 @@ options = {
         depCruiser.moduleWithDependencies("componentDefinition", "src/presentation/componentDefinition", [
             "src/model",
             "src/model/store/ui/uiSelectors.ts",
-            "src/presentation/filterBar",
+            "src/presentation/filterBar/*",
```

Comment:
Why is the star needed? Maybe you are missing some export for the index file of the filterBar directory?

Reply:
I need to import filterBar/types. Re-export usage leads to a circular dependency.

Reply:
Makes sense, in that case please be explicit like here: https://github.com/gooddata/gooddata-ui-sdk/blob/master/libs/sdk-ui-dashboard/.dependency-cruiser.js#L133-L134